src/popup/locales/cn.json

```diff
@@ -51,7 +51,7 @@
             "activity": "活动",
             "dashboard": "仪表板",
             "aeid": "❤币",
-            "coronaWallet": "SuperHero钱包",
+            "superheroWallet": "SuperHero钱包",
```

Comment:
Would be more flexible to rename it to something like `walletName` 😉

Reply:
Sure, I agree.